pkg/testingutil/pods.go

```diff
@@ -0,0 +1,22 @@
+package testingutil
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CreateFakePod creates a fake pod with the given pod name and component name
+func CreateFakePod(componentName, podName string) *corev1.Pod {
```

Comment:
this should be moved here https://github.com/openshift/odo/tree/master/pkg/kclient/fake

Reply:
hmm do we want it there? there are loads of Fake resource creation under pkg/testingutil like this https://github.com/openshift/odo/blob/master/pkg/testingutil/deployments.go

Reply:
Ok let's keep it in testingutil for now.